playbooks/roles/setup-caasp-workers/tasks/copy-certificates.yml

```diff
@@ -9,14 +9,25 @@
     dest: "/etc/pki/trust/anchors"
   register: _copiedcert
 
-- name: Refresh ca certs
-  become: True
-  command: update-ca-certificates
-  when: _copiedcert is changed
+- block:
```

Comment:
pretty cool, I didnt knew you could use a block to trigger using a when instead of having each task with the when!